moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -157,19 +155,16 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
       .reduce(
         (obj, item) =>
           Object.assign(obj, {
-            [item.subphase_name.toLowerCase()]:
-              item.subphase_name.charAt(0).toUpperCase() +
-              item.subphase_name.slice(1),
+            [item.subphase_id]: item.subphase_name,
```

Comment:
Echoing Frank and Mateo's similar comments that these kinds of changes are so nice to see!